Presentation/Activities/MainActivity.cs

```diff
@@ -28,17 +27,22 @@ private void LoadFragment(int id)
             switch (id)
             {
                 case Resource.Id.menu_home:
-                    fragment = new OpenWebTechnologyFragment();
+                    fragment = new OwtFragment();
```

Comment:
Please don't create new fragment everytime we need to change the view. It breaks the purpose of Fragments.

Reply:
I set the property RetainInstance in the fragment to true, and call the constructer in main activity.